enterprise/web.jsf/src/org/netbeans/modules/web/jsf/wizards/ManagedBeanPanelVisual.java

```diff
@@ -93,8 +95,13 @@ public ManagedBeanPanelVisual(Project proj) {
             }
         }
         Object[] scopes;
-        CdiUtil cdiUtil = proj.getLookup().lookup(CdiUtil.class);
-        isCDIEnabled = cdiUtil != null && cdiUtil.isCdiEnabled();
+        if(profile != null && (profile.isAtLeast(Profile.JAKARTA_EE_9_WEB) || profile.isAtLeast(Profile.JAKARTA_EE_9_FULL))){
```

Comment:
The check for the full profile should not be necessary. 
- 9 is it in both cases
- `WEB` is lower than `FULL`

```java
        if(profile != null && profile.isAtLeast(Profile.JAKARTA_EE_9_WEB)){
```